style/battle.css

```diff
@@ -630,7 +630,7 @@ License: GPLv2
 	background: #333333;
 	color: #CCCCCC;
 }
-.dark .battle-log .chat strong,
+.dark .battle-log .chat > strong,
```

Comment:
>How does light mode handle this problem?

It doesn't. It has the same problem, except the blue is slightly lighter.
>The really specific rules are my usual way?  

Well the last I did that you said [you're unhappy with them](https://github.com/Zarel/Pokemon-Showdown-Client/pull/879#discussion_r104277198) so that's why I tried something new this time.
Anyway, did this.